controllers/articles.js

```diff
@@ -510,6 +511,36 @@ const articles = {
         });
       }
     }
+  },
+
+  fbShare: async (req, res) => {
+    const { slug } = req.params;
+    const link = `${process.env.SERVER_ADDRESS}/articles/${slug}`;
+    res.status(200).send({
+      status: res.statusCode,
+      message: 'facebook share',
```

Comment:
@fridolinho I think the message here is not descriptive. Could you please work on it.